src/main/java/echoServer/Echoer.java

```diff
@@ -19,7 +23,7 @@ public Echoer(ServerSocketInterface serverSocket, ClientReadableFactory clientRe
     }
 
     public Socket startServer() throws IOException {
-        int port = 8080;
+        int port = 5000;
```

Comment:
Not a blocker for this card but you probably do want the flexibility of passing in the port through the command line args when running your server.

Reply:
Got it! I like the convenience of pushing the Intellij button but when Intellij runs out I will need that